test/test_latex.py

```diff
@@ -1,12 +1,13 @@
 """
 Tests for uqcsbot.scripts.cat
 """
-from .conftest import MockUQCSBot
-from .helpers import generate_message_object
+from .conftest import MockUQCSBot, TEST_CHANNEL_ID
 
 
 def test_latex(uqcsbot: MockUQCSBot):
-    message = generate_message_object("!latex test_memes")
-    uqcsbot.test_handle_event(message)
-    assert len(uqcsbot.test_posted_messages) == 1
-    assert uqcsbot.test_posted_messages[-1].text == "LaTeX render for \"test_memes\""
\ No newline at end of file
+    uqcsbot.post_message(TEST_CHANNEL_ID, "!latex test_memes")
+    messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
+    assert len(messages) == 2
+    print(messages)
+    assert messages[-1]["text"] == "LaTeX render for \"test_memes\""
+    assert len(messages[-1].get("attachments", [])) == 1
```

Comment:
new line

Reply:
also why not something like `messages[-1].get('attachments') is not None`?

Reply:
I want it to return exactly 1 attachment